src/KeraLua/NativeMethods.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Runtime.InteropServices;
 
 namespace KeraLua
 {
+	// typedef int (*lua_CFunction) (lua_State *L);
+	public delegate int LuaNativeFunction(LuaState luaState);

```

Comment:
Also, the idea is keep NativeMethods as agnostic as possible, 
We should pass IntPrt for consistence, and because of WP8 interop otherwise we wll break on UWP 😢